app/src/main/AndroidManifest.xml

```diff
@@ -23,10 +24,27 @@
         <activity android:name=".FeedbackActivity" />
         <activity android:name=".CouponsActivity" />
         <activity android:name=".PermissionsActivity" />
-        <activity android:name=".CouponPlainActivity"
-            android:label="Coupon detail activity"/>
-        <activity android:name=".FeedbackPlainActivity"
-            android:label="Feedback activity"/>
+        <activity
+            android:name=".CouponPlainActivity"
+            android:label="Coupon detail activity" />
+        <activity
+            android:name=".FeedbackPlainActivity"
+            android:label="Feedback activity" />
+
+        <!--  Built-in NearIT-UI Intent service, replace with your own if you want to handle notifications -->
```

Comment:
Add "For info, read the documentation section about (...)"

Reply:
@catt-stefano should I refer to the SDK documentation?

Reply:
You should refer to the NearIT-UI documentation regarding this particular feature, maybe not by link, but at least give enough info to let a user know that this is for a certain feature and it's not strictly required